### DISCUSSION TITLE
Add Tally Ho! Wallet and Customize Account Modal

### DISCUSSION THREAD

**Discussion by 0xAskar** [rainbow-me/rainbowkit#832] [Feedback]
This is a great library. Fantastic experience so far. However, two things

1) Can you add the (Tally Ho!)[https://tally.cash/) Crypto wallet. It's the first **decentralized** wallet and has tens of thousands of downloads? It has become many of ours primary wallet, so would be great to see.

2) Also, is there a way to customize our own Account Profile for the users signing into our site. I don't see an option in the library. There would be many modifications I would like to add without reconstructing my own (which I already have), but I would want to combine some of my features with yours?

Thanks for the help, and once again, for a fantastic module.

**Comment by am1993421** [comment]
To add to that.Tally Ho also is a community-owned and operated Web3 wallet.
The community already has over 70,000 member

**Comment by JVK-6935** [comment]
I support this proposal.

**Comment by iamchandrus** [comment]
I support this proposal!

**Comment by 0xzoz** [comment]
Thanks for the input @0xAskar 

Fortunately the work has been done for this in #824 🙂 

Excited to have apps using Rainbow with a Tally Ho option!
